src/platform/plugins/shared/unified_search/public/query_string_input/query_string_input.tsx

```diff
@@ -723,6 +729,7 @@ export class QueryStringInput extends PureComponent<QueryStringInputProps, State
     if (this.updateSuggestions.cancel) this.updateSuggestions.cancel();
     this.componentIsUnmounting = true;
     window.removeEventListener('resize', this.handleAutoHeight);
```

Comment:
Following my previous comment, this should be replaced by `window.removeEventListener('resize', this.handleResize);`

Reply:
Addressed in https://github.com/elastic/kibana/pull/228978/commits/2a1e4edc59fc39831bd10be36f8a68d55dd69718